opendut-util/opendut-auth/src/public/mod.rs

```diff
@@ -51,4 +51,5 @@ pub struct AuthData {
     pub email: String,
     pub groups: Vec<String>,
     pub roles: Vec<String>,
+    pub subject: String,
```

Comment:
Add a comment here, that the subject contains the key identifier of the identity provider for this user. This is basically the user identity in keycloak.

Reply:
Done